documentation/adding-new-rtypes.md

```diff
@@ -47,7 +57,15 @@ a minimum.
 
 -   Add the capability to the file `dnscontrol/providers/capabilities.go` (look for `CanUseAlias` and add
     it to the end of the list.)
--   Add this feature to the feature matrix in `dnscontrol/build/generate/featureMatrix.go` (Add it to the variable `matrix` then add it later in the file with a `setCap()` statement.
+-   Run stringer to auto-update the file `dnscontrol/providers/capability_string.go`
+
+```shell
+pushd; cd providers;
+stringer -type=Capability
```

Comment:
"go generate" should be sufficient.  (though you do need to be in the correct subdirectory)

```
~/git/dnscontrol/providers$ ls -l capability_string.go
-rw-r-----  1 tlimoncelli  staff  1365 Mar 14 17:48 capability_string.go
~/git/dnscontrol/providers$ rm capability_string.go
~/git/dnscontrol/providers$ go generate
~/git/dnscontrol/providers$ ls -l capability_string.go
-rw-r-----  1 tlimoncelli  staff  1365 Mar 14 17:49 capability_string.go
```

Reply:
can't think why that didn't work for me, or that I might not have tried that. Documented both.